src/languages/duckdb/duckdb.keywords.ts

```diff
@@ -0,0 +1,155 @@
+export const keywords: string[] = [
```

Comment:
Missing some keywords. For example:

COALESCE, COLLATION, etc.

```sql
 SELECT
   UPPER(keyword_name) AS keyword_name
 FROM
   duckdb_keywords ();
```

Reply:
Based on this feedback the keywords are sourced from:

```sql
    SELECT upper(keyword_name)
    FROM duckdb_keywords()
    WHERE keyword_category = 'reserved'
    ORDER BY keyword_name
```